qualang_tools/plot/Fitting/fitting.py

```diff
@@ -511,22 +523,22 @@ def transmission_resonator_spectroscopy(
         if guess is not None:
             for key in guess.keys():
                 if key == "f":
-                    f0 = guess[key] / x_normal
+                    f0 = float(guess[key]) / x_normal
                 elif key == "k":
-                    width0 = guess[key] / x_normal
+                    width0 = float(guess[key]) / x_normal
                 elif key == "offset":
-                    v0 = guess[key] / y_normal
+                    v0 = float(guess[key]) / y_normal
                 else:
                     raise Exception(
                         f"The key '{key}' specified in 'guess' does not match a fitting parameters for this function."
                     )
-
+        # Print the initial guess if verbose=True
         if verbose:
             print(
-                f"Initial guess:\n"
-                f" f = {f0 * x_normal}, \n"
-                f" kc = {(peak - v0)/width0 * y_normal / x_normal}, \n"
-                f" k = {width0 * x_normal}, \n"
+                f"Initial guess:\n "
+                f" f = {f0 * x_normal}, \n "
+                f" kc = {(peak - v0) * (width0 * x_normal) * y_normal}, \n "
```

Comment:
Shouldn't it be `{(peak - v0) / (width0 * x_normal) * y_normal}` instead?

Reply:
since it is kc/k in order to get kc I need to do kc/k*k

Reply:
Ok I believe you, it is just that it was the inverse before the commit this is why I was confused

Reply:
Yeah I know.. I checked it but double check is always good :)

Reply:
To be sure we need to implement unit tests, do you know how to do it?

Reply:
no.. can I see it somewhere or do you want to hop on a call?